testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -75,6 +76,7 @@
   private String tableName;
   private static String authToken;
   private String host;
+  private String restUrlBase;
```

Comment:
This is added to help with merging to Stargate V2 where we need(ed) to override things; may over time change usage here.